cmake/packaging.cmake

```diff
@@ -64,7 +71,11 @@ endif()
 
 set (VS "") # VisualStudio
 if ( MSVC )
-    if     ( MSVC_VERSION STREQUAL 1900 )
+    if     ( MSVC_VERSION STREQUAL 2100 )
```

Comment:
Did you try this? According to the CMake documentation the values are different:

https://cmake.org/cmake/help/latest/variable/MSVC_VERSION.html

Reply:
Well spotted.  I totally made that up and I have it wrong!

I am aware of this.  These values are the _MSC_VER and you'll see the code in src/version.cpp reports 2017 and 2019 correctly.  It doen't feel correct to use STREQUAL.  The current version of 2017 has _MSC_VER at 1916, so I think we'll need an integer value comparison.